tests/test_manifester.py

```diff
@@ -66,11 +74,17 @@
     ],
 }
 
-sub_allocations_response = {
+SUB_ALLOCATIONS_RESPONSE = {
     "body": [
         {
-            "uuid": f"{uuid.uuid4().hex}",
-            "name": "test_user-" + "".join(random.sample(string.ascii_letters, 8)),
+            "uuid": f"{SUB_ALLOCATION_UUID}",
+            "name": f"{MANIFEST_DATA['username_prefix']}-"
+            + "".join(random.sample(string.ascii_letters, 8)),
```

Comment:
minor performance gain by switching to choices
```suggestion
            + "".join(random.choices(string.ascii_letters, k=8)),
```

```python
In [3]: %timeit random.sample(string.ascii_letters, 8)
2.52 µs ± 25.7 ns per loop (mean ± std. dev. of 7 runs, 100,000 loops each)

In [4]: %timeit random.choices(string.ascii_letters, k=8)
881 ns ± 10.3 ns per loop (mean ± std. dev. of 7 runs, 1,000,000 loops each)
```